ena-submission/src/ena_deposition/call_loculus.py

```diff
@@ -146,19 +148,32 @@ def get_group_info(config: Config, group_id: int) -> Group:
 
 
 def fetch_released_entries(config: Config, organism: str) -> Iterator[dict[str, Any]]:
-    """Get sequences that are ready for release"""
-
+    """
+    Stream zstd-compressed NDJSON and yield filtered dicts.
+    """
     url = f"{organism_url(config, organism)}/get-released-data"
+    params = {"compression": "ZSTD"}
+    headers = {
+        "Accept": "application/x-ndjson",
```

Comment:
I'm a bit uncertain about the change. You point out that it doesn't make it any faster. That's because when things are on the same network, or in fact on the same machine sometimes, data transfer is really cheap/fast. On the other hand compressing and decompressing stuff is expensive. I understand that "it definitely doesn't make things slower and is working" but it will definitely increase CPU usage, and for no detectable performance increase so far. IMO we know that this will increase resource usage and we don't know it will make things any faster so we shouldn't merge